generated/api/v1/alert_service.swagger.json

```diff
@@ -1451,16 +1451,13 @@
           }
         },
```

Comment:
Please ignore these changes. These changes are from a bug in swagger generation process